cmd/localapiclient/localapiclient.go

```diff
@@ -1,87 +1,82 @@
 package localapiclient
 
 import (
-	"bytes"
-	"errors"
+	"context"
 	"fmt"
-	"log"
+	"io"
 	"net/http"
 	"time"
 
 	"tailscale.com/ipn/localapi"
 )
 
-// LocalAPIResponseWriter substitutes for http.ResponseWriter in order to write byte streams directly
-// to a receiver function in the application.
-type LocalApiResponseWriter struct {
-	headers http.Header
-	body    bytes.Buffer
-	status  int
+// Response represents the result of processing an http.Request.
+type Response struct {
```

Comment:
I think the consumer of the `localapiclient` package doesn't necessarily care that this is a fake ResponseWriter, they just want to able to use it as a `Response` that gives them the info they need. Thus the suggested name and comment change.